mozilla/foundation.mozilla.org
# added UL to tips to protect self

Closes #7700 
Related PRs/issues #7707 

This PR was made to allow the users to add a UL in the "tips to protect yourself" field in the CMS. That way they can just create a bullet point for every "tip" and it will render correctly as a UL on the front end with the changes made in #7707.

**Link to sample test page**: https://foundation-s-7700-updat-yy2imi.herokuapp.com/cms/pages/23/edit/

**Steps to test:**

1. Visit the CMS using the link above
2. Scroll down to the "Tips to protect yourself" section"
3. The section should now have 4 options: Bold, Italic, List, and Link.
4. Please feel free to test out adding tips to the section and if everything is working as expected, testing is complete!
**Note**: Please note however that since the front end changes have not been merged into master yet, the changes will not appear on the front end.
